launch.go

```diff
@@ -72,6 +73,32 @@ func init() {
 	client = retryablehttp.NewClient()
 }
 
+/* has HTTP or HTTPS protocol
+targetUrl => URL
+*/
+func hasHTTPProtocol(targetUrl *url.URL) bool {
+	return strings.Contains(targetUrl.Scheme, "http") || strings.Contains(targetUrl.Scheme, "https")
```

Comment:
golang source code says below. (ref: https://golang.org/src/net/url/url.go?#L428)

> // Maybe rawurl is of the form scheme:path.
> // (Scheme must be [a-zA-Z][a-zA-Z0-9+-.]*)
> // If so, return scheme, path; else return "", rawurl.

Therefore I think we should check only whether `targetUrl.Scheme` is http/https or not.

Reply:
I fixed.